spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+RSpec.feature 'User show page', type: :feature do
+  let(:user) { create(:user) }
+  let!(:posts) { create_list(:post, 5, author: user) }
+  scenario 'displays user information and interactions' do
+    visit user_path(user)
+    expect(page).to have_selector('.userImg', visible: false, count: 1)
+    expect(page).to have_css('.userName', text: user.name)
+    expect(page).to have_css('.Upostcount', text: "Number of Posts: #{user.posts.count}")
+    expect(page).to have_css('.bioText', text: user.bio)
+    expect(page).to have_button('See all posts', visible: true)
+    click_button('See all posts')
+    expect(page).to have_current_path(user_posts_path(user))
+  end
+  scenario 'displays the user\'s first 3 posts with static content' do
+    visit user_path(user)
+    rec_posts = all('.recPost')
+    expect(rec_posts.count).to eq(3)
+    user.posts.first(3).each_with_index do |_post, index|
+      actual_post_title = rec_posts[index].find('h3 strong').text.strip
+      expected_post_title = 'Post: Sample Post Title'.strip
```

Comment:
* I get this error when i try to run your tests , kindly look into the said error .
*  Below is a screenshot of the error: 
![image](https://github.com/chimwemwe007/blog-app/assets/52497246/f80fecb4-3e54-461e-b9ed-8be79e657577)